templates/app-ts/src/app.ts

```diff
@@ -18,17 +18,18 @@ const app: FastifyPluginAsync<AppOptions> = async (
   // those should be support plugins that are reused
   // through your application
   void fastify.register(AutoLoad, {
-    dir: path.join(__dirname, 'plugins'),
+    dir: join(__dirname, 'plugins'),
     options: opts
   })
 
   // This loads all plugins defined in routes
   // define your routes in one of these
   void fastify.register(AutoLoad, {
-    dir: path.join(__dirname, 'routes'),
+    dir: join(__dirname, 'routes'),
     options: opts
   })
 
 };
 
 export default app;
+export { app }
```

Comment:
why this change?

Reply:
Fair point. This is my pet peeve with the lack of named exports to go with the default, so for convenience of writing a test I added it, but if you prefer having just a default, I can remove it.

Reply:
Nice ok. Maybe in the test you can also check the default export?

Reply:
Done.